examples/language/llama2/requirements.txt

```diff
@@ -1,4 +1,4 @@
-colossalai>=0.3.0
+colossalai==0.3.2
```

Comment:
I don't know whether fixing the version of colossalai is appropriate. Some bugs will be fixed in the next released version.